### ISSUE TITLE
How to solve the problems when inplanes != planes

### ISSUE THREAD

**Issue by Senwang98** [backseason/SCNet#15]
Hi,
If I want to use scconv which inplanes = 64, planes = 3, there is an error in code:
`out = torch.sigmoid(torch.add(identity, F.interpolate(self.k2(x), identity.size()[2:]))) # sigmoid(identity + k2)`
